specs/codecs/dag-cosmos/crypto_types.md

```diff
@@ -0,0 +1,29 @@
+# Tendermint and Cosmos Cryptographic Types
+
+```ipldsch
+# Signatures in Tendermint are raw bytes representing the underlying signature
+type Signature bytes
+
+# Public key
+type PubKey bytes
+
+# Private key
+type PrivKey bytes
+
+# Hashes from leaf's sibling to a root's child
+type Aunts [Hash]
+
+# Proof represents a Merkle proof.
+# NOTE: The convention for proofs is to include leaf hashes but to
+# exclude the root hash.
+# This convention is implemented across IAVL range proofs as well.
+# Keep this consistent unless there's a very good reason to change
+# everything.  This also affects the generalized proof system as
+# well.
+type Proof struct {
+	Total    Int  # Total number of items.
+	Index    Int # Index of item to prove.
+	LeafHash Hash # Hash of item value.
+	Aunts    Aunts # Hashes from leaf's sibling to a root's child.
```

Comment:
Is there any utility in _also_ presenting an `AuntsLinks` or `AuntsCIDs` in the data model where you reinterpret the list of hashes as CIDs that can be navigable? Or is this purely a useful construct for the proof itself and navigation isn't really helpful at this point?

Reply:
I think it would be valuable to make it navigable, will make this update.